lib/DateTime.vala

```diff
@@ -68,6 +99,16 @@ namespace Granite.DateTime {
     }
 
     /**
+     * Compare two {@link GLib.DateTime} and @return true if they occur on the same day of the same year
+     */
+    public bool is_same_day (GLib.DateTime day1, GLib.DateTime day2) {
```

Comment:
All of the points I made above apply here including, `static`, documentation, using proper `@param`'s and `@return`'s.
You should write here that it comapares the year as well as the day but nothing more.